.stylelintrc.json

```diff
@@ -22,6 +23,7 @@
     ],
     "declaration-property-value-disallowed-list": {
       "text-align": ["right", "left"]
-    }
+    },
+    "spartacus/comment-no-loud": true
```

Comment:
It would be nice to be more expressive about the fact that this rule only applies to (s)css. Either by a more descriptive name or by passing in a file pattern (i.e. `"spartacus/comment-no-loud": 'css'`)